packages/blockchain-wallet-v4-frontend/src/modals/Onboarding/UpgradeNowSilver/template.success.tsx

```diff
@@ -0,0 +1,361 @@
+  display: flex;
+  max-width: 480px;
+  background-color: ${(props) => props.theme.white};
+  padding: 40px 40px 0 40px;
+`
+const RowItemTitle = styled(Text)`
+  color: ${(props) => props.theme.grey900};
+  font-size: 16px;
+  padding-left: 16px;
+  font-weight: 600;
+  line-height: 150%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  flex: 1;
+`
+
+const RowItemWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  flex: 1;
+`
+
+const RowItemSubTitle = styled(Text)`
+  color: ${(props) => props.theme.grey600};
+  font-size: 14px;
+  padding-left: 16px;
+  font-weight: 500;
+  line-height: 150%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  flex: 1;
+`
+const RowItemTitleWhite = styled(RowItemTitle)`
+  color: ${(props) => props.theme.white};
+`
+const RowItemSubTitleWhite = styled(RowItemSubTitle)`
+  color: rgba(255, 255, 255, 0.72);
+`
+
+const MainContent = styled.div``
+
+const StatusCartridge = styled(GreyCartridge)`
+  display: flex;
+  flex-direction: row;
+  > span {
+    padding: 3px 8px;
+  }
+`
+const StatusCartridgeBlue = styled(BlueCartridge)`
+  display: flex;
+  flex-direction: row;
+  background: rgba(255, 255, 255, 0.24);
+  > span {
+    padding: 3px 8px;
+  }
+`
+
+const UpgradeContainer = styled.div<{ second?: boolean }>`
+  border: 1px solid ${(props) => props.theme.grey000};
+  box-sizing: border-box;
+  border-radius: 16px;
+  margin: 16px 40px;
+  ${(props) =>
+    props.second &&
+    css`
+      border: 1px solid ${(props) => props.theme.blue400};
+      background-color: ${(props) => props.theme.blue600};
+    `}
+`
+
+const Disclaimer = styled.div`
+  display: flex;
+  justify-content: center;
+  text-align: center;
+  margin-bottom: 20px;
+`
+
+const UpgradeRow = styled.div`
+  display: flex;
+  padding: 16px 24px;
+`
+const UpgradeRowWithBorder = styled(UpgradeRow)`
+  border-bottom: 1px solid ${(props) => props.theme.grey000};
+`
+const UpgradeRowWithBlueBorder = styled(UpgradeRow)`
+  border-bottom: 1px solid ${(props) => props.theme.blue400};
+`
+
+const CloseIconContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 32px;
+  height: 32px;
+  border-radius: 50%;
+  background: ${(props) => props.theme.grey000};
+  backdrop-filter: blur(54.3656px);
+  > span {
+    justify-content: center;
+  }
+`
+
+type Props = OwnProps & SuccessStateType
+
+const Template: React.FC<Props> = (props) => {
+  const { identityVerificationActions, modalActions, userTiers } = props
+  const startVerification = useCallback(() => {
+    modalActions.closeModal(ModalName.TRADING_LIMITS_MODAL)
+    modalActions.closeModal(ModalName.UPGRADE_NOW_MODAL)
+    identityVerificationActions.verifyIdentity({
+      needMoreInfo: false,
+      origin: 'Settings'
+    })
+  }, [identityVerificationActions, modalActions])
+  if (!Array.isArray(userTiers)) {
+    return null
+  }
+
+  return (
+    <Wrapper>
```

Comment:
If you want you could use FlyoutContainer, FlyoutHeader, and FlyoutContent here instead of having to create all new components for this stuff.

Reply:
Will use some, this page is a bit specific we might need to improve such components to be able to render all on proper way